### PR TITLE
removes headpatting

### PR DESCRIPTION
**Pull request by imsxz**
## About The Pull Request
reverts #53855 

## Why It's Good For The Game

it was never meant to be this way

## Changelog

:cl: imsxz
del: no headpatting
/:cl:

### CODE REVIEW

**Comment by redsunoverparadise** [comment]
1984

**Comment by imsxz** [comment]
i brought headpatting into this world, it's my duty to take it out

**Comment by Malgover** [comment]
based

**Comment by SweptWasTaken** [comment]
> 
> 
> i brought headpatting into this world, it's my duty to take it out

Damn you get banned from FF14 for 3 days and you're already recovering your health. Incredible!

**Comment by imsxz** [comment]
dont remind me

**Comment by tralezab** [comment]
Closing at 24 hours unless another maintainer wants this, I'm not removing headpats until someone headpats my static (please stop ingoring me ca/tg/irls it hurts more than you could know)

**Comment by imsxz** [comment]
u ignored my ping on discord but came crawling out of the woodwork to slap a "do not merge" on my pr

i thought what we had was special

**Comment by tralezab** [comment]
Also may want to actually justify your change in the 24 hours you have if you're going to convince another maintainer to keep the pr alive!!

**Review by NamelessFairy — COMMENTED** [review]
Imsxz requested that I review their PR on discord, code checks out 🤷

File: code/modules/mob/living/carbon/carbon_defense.dm
Comment (Paxilmaniac) [inline code/modules/mob/living/carbon/carbon_defense.dm:448]: It seems you have made a mistake here and accidentally deleted headpatting when making the pull request.

Comment (imsxz) [reply code/modules/mob/living/carbon/carbon_defense.dm:448]: nyoops!

**Review by wesoda25 — APPROVED** [review]
looks good 👍

**Comment by itseasytosee** [comment]
I appreciate the attempted redemption arc.

**Review by itseasytosee — APPROVED** [review]

**Comment by Mothblocks** [comment]
Over my dead fucking body

**Comment by imsxz** [comment]
24 hour rule hello?

**Comment by Mothblocks** [comment]
There is no 24 hour rule for closing

**Comment by tralezab** [comment]
pfffhaaha imsxz i told you to CONVINCE someone else, not get it closed before 24 hours

**Comment by Mothblocks** [comment]
this pr got me out of bed

**Comment by imsxz** [comment]
> 
> 
> pfffhaaha imsxz i told you to CONVINCE someone else, not get it closed before 24 hours

why dont u convince a girl to date u

**Comment by Bawhoppen** [comment]
Imsxz redemption arc almost happened

**Pull request closed without merging**
